docs/usage/setup.md

```diff
@@ -2,40 +2,38 @@
 
 Firstly you need to open the browser at the link <http://localhost:8091> and edit the settings for Zwave, MQTT and the Gateway.
 
-## Zwave
+## General
+
+- **Log enabled**: Enable logging
+- **Log level**: Set the zwave-js Library log level
```

Comment:
This are not zwave-js logging settings, this are the zwavejs2mqtt settings

Reply:
My bad, didn't even noticed.